.tool-versions

```diff
@@ -1,2 +1,2 @@
 elixir 1.10.4-otp-23
-erlang 23.0.3
+erlang 23.1.1
```

Comment:
Can we update to `23.2.3` instead?

And while we're at it: update Elixir to `1.11.3-otp-23` and update the test matrix in the gh actions workflow?